sdk/textanalytics/ai-text-analytics/src/detectLanguageResult.ts

```diff
@@ -55,13 +55,7 @@ export function makeDetectLanguageErrorResult(
 
 function primaryLanguage(languages: DetectedLanguage[]): DetectedLanguage {
   const sorted = languages.slice(0).sort((a, b) => {
-    if (a.score === undefined) {
```

Comment:
wonder if this was able to be undefined in a previous schema, since it seems bonkers we put this logic here on its own